### PR TITLE
Fix plugin failing when single types existed

### LINKED ISSUE #122: Meilisearch can't work with Strapi single type content
```
node v14.16.0
strapi v3.5.4
strapi-plugin-meilisearch v0.2.0
ubuntu v20.04
```

Hi there,

I spotted an issue inside numberOfRowsInCollection function.
Indeed, I get `strapi.services[collection].count is not a function`
But if I the function returns 1 it works.

The problem comes from Strapi single type content.
In my case, I had an Home Page single type content.
Once removed, it's perfectly fine.

It's only because, you can't count on a Strapi single type content.

Nevertheless, Meilisearch doesn't should fetch this kind of "collection".

### PR DESCRIPTION
**Pull request by bidoubiwa**
Before we decide how to handle `single types` in MeiliSearch this PR fixes the issue by removing single types from the types selection in the plugin.

Fixes #122 

open issue: #124

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
can you remove `playground/.tmp/data.db` file?

**Comment by bidoubiwa** [comment]
It is needed for the tests! Its a small database in SQLITE containing the collections used during testing

**Review by curquiza — APPROVED** [review]

File: controllers/meilisearch.js [hunk 1]
```diff
@@ -119,8 +119,18 @@ async function fetchRowBatch({ start, limit, collection }) {
   })
 }
 
+function getCollectionTypes() {
+  const services = strapi.services
+  return Object.keys(strapi.services).filter(type => {
```
Comment (bidoubiwa) [inline controllers/meilisearch.js:124]:
```suggestion
  return Object.keys(services).filter(type => {
```

**Review by curquiza — APPROVED** [review]

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**